jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/RangeSlidersTests.java

```diff
@@ -92,13 +85,24 @@ public void thumbLabelRangeSliderTest() {
 
         thumbLabelRangeSlider.is().rightThumbSize(16);
         thumbLabelRangeSlider.is().leftThumbSize(16);
+
+        adjustableRangeSlider.show();
+        adjustableRangeSlider.is().thumbLabelDisplayed();
+        thumbLabelControl.click();
+        adjustableRangeSlider.has().noThumbLabels();
     }
 
-    @Test(description = "Check ranger slider's color")
-    public void thumbRangeSliderColor(){
-        colorRangerSlider.show();
-        colorRangerSlider.setRangerSliderColor("244", "67", "54");
-        waitCondition(() -> colorRangerSlider.trackColor().equals(Colors.RED.value()));
-        colorRangerSlider.has().trackColor(Colors.RED.value());
+    @Test(description = "Test checks range slider's color", dataProvider = "trackRangeSliderColorTestData", dataProviderClass = RangeSliderTestsDataProvider.class)
+    public void trackRangeSliderColor(String red, String green, String blue){
+        colorRangeSlider.show();
+        colorPicker.get(1).clear();
+        colorPicker.get(1).sendKeys(red);
+        colorPicker.get(2).clear();
+        colorPicker.get(2).sendKeys(green);
+        colorPicker.get(3).clear();
+        colorPicker.get(3).sendKeys(blue);
```

Comment:
If we'll use an element of the colorPicker class, then perhaps this section will be simplified?